### PR TITLE
Fix folder copying to sibling folder with almost the same name

### PR DESCRIPTION
**Pull request by tsvietOK**
To reproduce this bug create folder with name "Some Folder" and another sibling folder with name "Some Folder 1".  Try to copy or move first folder in second.

### CODE REVIEW

**Comment by duke7553** [comment]
@tsvietOK Hi thanks for this PR.

The original issue that this block of code tried to solve was users copying a folder into itself. Does this change still prevent all instances of a destination folder being a subfolder of the source folder?

image

**Comment by tsvietOK** [comment]
@duke7553 hmmm, your right, i didn't test this case. I have to find another workaround.

**Comment by duke7553** [comment]
@tsvietOK I'll mark this as a draft for now then.

**Comment by tsvietOK** [comment]
Taken from https://stackoverflow.com/questions/5617320/given-full-path-check-if-path-is-subdirectory-of-some-other-path-or-otherwise
@duke7553  what do you think about it?

**Comment by duke7553** [comment]
@tsvietOK That looks pretty good to me

**Comment by tsvietOK** [comment]
@duke7553 do you have troubles with folder copying?

**Comment by duke7553** [comment]
@tsvietOK Do you want me to test this now? I can mark this as ready for review.

**Comment by tsvietOK** [comment]
@duke7553 If your copy function working fine, of course.

**Comment by tsvietOK** [comment]
@duke7553 did you tested it?

File: Files/Interacts/Interaction.cs [hunk 1]
```diff
@@ -888,7 +889,7 @@ public async Task PasteItems(DataPackageView packageView, string destinationPath
             {
                 if (item.IsOfType(StorageItemTypes.Folder))
                 {
-                    if (destinationPath.Contains(item.Path, StringComparison.OrdinalIgnoreCase))
```
Comment (yaichenbaum) [inline Files/Interacts/Interaction.cs:891]: Instead of adding a full class and removing this line, would it work to change `Contains` to `Equals`?

Comment (tsvietOK) [reply Files/Interacts/Interaction.cs:891]: "D:/Some Folder" copy to "D:/Some Folder/Some Folder another/...". Second folder is subfolder for first folder. But paths are not equals

Comment (yaichenbaum) [reply Files/Interacts/Interaction.cs:891]: Ok, thanks for explaining.

**Comment by yaichenbaum** [comment]
@tsvietOK Can you merge with the latest from `develop`?

**Comment by yaichenbaum** [comment]
Before merging, this should be tested to make sure that the changes work in release mode.

**Comment by tsvietOK** [comment]
@yair100 Part with subfolder detection working fine for me. Another thing, we currently don't  supporting Windows's path length limit(as i understand 260 characters)

**Review by yair100 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@tsvietOK I would think that is a different issue, right?

**Comment by tsvietOK** [comment]
@yair100 Yes, your right

**Pull request merged**